src/clientlayers/ExceptionRequest.jl

```diff
@@ -10,14 +10,15 @@ using ..IOExtras, ..Messages, ..Exceptions
 Throw a `StatusError` if the request returns an error response status.
 """
 function exceptionlayer(handler)
-    return function exceptions(stream; status_exception::Bool=true, logerrors::Bool=false, kw...)
-        res = handler(stream; logerrors=logerrors, kw...)
+    return function exceptions(stream; status_exception::Bool=true, timedout=nothing, logerrors::Bool=false, logtag=nothing, kw...)
```

Comment:
IIUC, `timedout` kwarg is strictly internal thing, the user shouldn't provide their own `TimedOut` struct to a `HTTP.get`? If that is the case, we might want to call it `_timedout` just to make it clearer that this is an internal thing.